kotlintest-assertions/src/main/kotlin/io/kotlintest/properties/Gen.kt

```diff
@@ -48,7 +48,7 @@ class BigIntegerGen(maxNumBits: Int) : Gen<BigInteger> {
  * from across the entire integer range.
  */
 @higherkind
-interface Gen<T> : GenOf<T> {
+interface Gen<T> {
```

Comment:
Is this intentional this time?
I think this change and some other files are breakin arrow, aren't them?